Makefile

```diff
@@ -83,11 +80,17 @@ testrace: prep
 	go test -tags='$(BUILD_TAGS)' -race $(TEST) $(TESTARGS) -timeout=$(EXTENDED_TEST_TIMEOUT) -parallel=20
 
 cover:
+	for dir in $(SUBDIRS); do \
+		$(MAKE) -C $$dir; \
+	done
 	./scripts/coverage.sh --html
 
 # vet runs the Go source code static analysis tool `vet` to find
 # any common errors.
 vet:
+	for dir in $(SUBDIRS); do \
```

Comment:
Might be worth also doing this for loop in the `fmt` target.

Reply:
`fmt` uses find to get files and as such works across all of the modules. It's in the API dir Makefile in case you want to run it _only_ in there but at the top level will do all files.

Reply:
Ah missed that.